packages/buttons/src/styled/StyledButtonGroup.spec.tsx

```diff
@@ -37,10 +36,11 @@ describe('StyledButtonGroup', () => {
       </StyledButtonGroup>
     );
 
+    /* stylelint-disable */
     expect(getByTestId('group-button')).toHaveStyleRule('position', 'relative', {
-      modifier: css`
+      modifier: (`
```

Comment:
is the `css` removal intended?

Reply:
Yes, there were stylelint issues when this got moved to TypeScript that I couldn't remove. Also, the tests still pass so I don't think they are necessary in this context.